server/@types/viewModels/index.d.ts

```diff
@@ -216,6 +216,8 @@ declare module 'viewModels' {
   export interface Action {
     title: string
     href: string
-    dataQa?: string
+    id?: string
```

Comment:
I've added `id` and `attributes` so that the Action Card component can be used with the same types of properties that you would with any other GDS component (and therefore `data-qa` doesn't need to be it's own property)

I've also added a property `render-if` (happy to take suggestions on better/different name?) which can be used to conditionally render the action item in the list (see how it is used in the template below)